python/ribasim/ribasim/node_types/flow_boundary.py

```diff
@@ -15,6 +17,14 @@ class Config:
         dtype = PydanticModel(models.FlowBoundaryStatic)
 
 
+class TimeSchema(pa.SchemaModel):
+    class Config:
+        """Config with dataframe-level data type."""
+
+        dtype = PydanticModel(models.FlowBoundaryTime)
+        coerce = True  # this is required, otherwise a SchemaInitError is raised
```

Comment:
Was that the solution for the empty dataframe problem we encountered yesterday?

Reply:
No it wasn't, I wanted an empty `FlowBoundary / static` but I added a dummy node in there. In the end that was fine since I needed to test having both static and dynamic nodes. The problem persists though, I will post about it in the dev channel.